src/main/java/org/apache/commons/lang3/function/Consumers.java

```diff
@@ -46,4 +46,17 @@ private Consumers() {
         // No instances.
     }
 
+    /**
+     * Applies the given {@link Consumer} action to the object if the object is not null. If the object is null, it does
+     * nothing.
+     *
+     * @param object the object to be consumed.
+     * @param consumer the consumer to consume.
+     * @param <T> the type of the argument the consumer accepts.
+     */
+    public static <T> void acceptIfNotNull(final T object, final Consumer<T> consumer) {
```

Comment:
- The name can just be `accept`.
- Add a Javadoc since tag.
- Add null-check for consumer, see `Suppliers.get()`